tools/update-release-names.sh

```diff
@@ -1,8 +1,9 @@
 #!/bin/sh
 
 if [ x"$4" == x ]; then
-  echo "Usage:   $0 version-code current-version beta-version alpha-version"
+  echo "Usage:   $0 version-code current-version beta-version alpha-version [current-branch] [beta-branch] [alpha-branch]"
```

Comment:
Should we update the `version-code` parameter naming here for `current-alpha-version-code` to make sure the people who run this script ;) use the right starting version code number? And then let the script make that number plus one. I think it's easier to just look up for the version code in `alpha` and let the script generate the next 3 codes than having the user have to remember they need to look up and then add one more. Maybe an automated check would be ideal (pull latest alpha, check versionCode....), but that's too much to ask, good for a trial though

Reply:
Good idea, but I think I'll do something differently: I'll check the version code in `develop` and compare with the one entered by the user, if it's not greater than the one we have in `develop` I'll make the script fail.

Reply:
💯

Reply:
Updated in 0207ce2